services/management/checks.go

```diff
@@ -103,20 +105,27 @@ func (s *ChecksAPIService) ListSecurityChecks(ctx context.Context, req *manageme
 			Disabled:    disabled,
 			Summary:     c.Summary,
 			Description: c.Description,
+			Interval:    convertInterval(c.Interval),
 		})
 	}
 
 	return &managementpb.ListSecurityChecksResponse{Checks: res}, nil
 }
 
-// ChangeSecurityChecks enables/disables Security Thread Tool checks by names.
+// ChangeSecurityChecks enables/disables Security Thread Tool checks by names or changes its execution interval.
 func (s *ChecksAPIService) ChangeSecurityChecks(ctx context.Context, req *managementpb.ChangeSecurityChecksRequest) (*managementpb.ChangeSecurityChecksResponse, error) {
 	var enableChecks, disableChecks []string
+	changeIntervalParams := make([]checks.ChangeIntervalParams, 0, len(req.Params))
 	for _, check := range req.Params {
 		if check.Enable && check.Disable {
 			return nil, status.Errorf(codes.InvalidArgument, "Check %s has enable and disable parameters set to the true.", check.Name)
 		}
 
+		changeIntervalParams = append(changeIntervalParams, checks.ChangeIntervalParams{
```

Comment:
We should update interval only if it's passed.

Reply:
let's use `map` instead of `checks.ChangeIntervalParams`, so you don't need to loop again.

Reply:
done